app/components/course-objective-manager.js

```diff
@@ -106,28 +105,44 @@ export default Component.extend({
     }
   }),
 
+  course: computed('courseObjective.courses.[]', async function () {
+    const courseObjective = this.get('courseObjective');
+    if (! courseObjective) {
+      return [];
```

Comment:
kind of a weird return value for "does not exists". may i suggest `null` instead?

Reply:
I didn't want to mess with the existing functionality too much, took this from https://github.com/ilios/frontend/blob/master/app/components/course-objective-manager.js#L117.  I had a note to myself to refactor after they are moved to common, but I just created an issue for more general use https://github.com/ilios/frontend/issues/4076

Reply:
fair enough. empty array evaluates falsy, so this should be alright. let's fix it later then.